### ISSUE TITLE
Records with long IDs don't load in basket or editor

### ISSUE THREAD

**Issue by jbukhari** [dag-hammarskjold-library/dlx-rest#384]

**Comment by aaronhelton** [comment]
There is an issue here, but it's unlikely to cause problems with a proper transition from Horizon.

**Comment by viola-v** [comment]
For ISS to check, not exactly sure what the issue is...

**Comment by viola-v** [comment]
@aaronhelton Could you check if this can be closed?

**Comment by aaronhelton** [comment]
@jbukhari I don't think this is still an issue, but if it is, nobody has reported anything related to it in the last year. Maybe we can close it?

**Comment by jbukhari** [comment]
The database apparently has some issues dealing with integers representing a size above 8 bytes. Also, JavaScript can automatically convert long ints (more than 21 digits) to a scientific notation string.

In real life IDs will never be this large. This was only affecting test data. Can close the issue acknowledging known constraints.
